docs/content.zh/release-notes/flink-1.20.md

```diff
@@ -0,0 +1,434 @@
+This allows the connector to dynamically determine parallelism based on the actual partitions with dynamic partition pruning.
+Additionally, we have introduced a new configuration option - `table.exec.hive.infer-source-parallelism.mode` to enable
+users to choose between static and dynamic inference modes for source parallelism. By default, the mode is set to `dynamic`.
+Users may configure it to `static` for static inference, `dynamic` for dynamic inference, or `none` to disable automatic parallelism inference altogether.
+
+It should be noted that in Flink 1.20, the previous configuration option `table.exec.hive.infer-source-parallelism` has been marked as deprecated, 
+but it will continue to serve as a switch for automatic parallelism inference until it is fully phased out.
+
+#### Promote Unified Sink API V2 to Public and Deprecate SinkFunction
+
+##### [FLINK-35378](https://issues.apache.org/jira/browse/FLINK-35378)
+
+`TwoPhaseCommitSinkFunction`, `SocketClientSink`, `RichSinkFunction`, `PrintSinkFunction` and `DiscardingSink`
+have been deprecated in favor of the new `org.apache.flink.streaming.api.functions.sink.v2.*` interfaces.
+
+#### Support for HTTP connect and timeout options while writes in GCS connector
+
+##### [FLINK-32877](https://issues.apache.org/jira/browse/FLINK-32877)
+
+The current GCS connector uses the gcs java storage library and bypasses the hadoop gcs connector 
+which supports multiple http options. There are situations where GCS takes longer to provide a response
+for a PUT operation than the default value.
+
+This change will allow users to customize their connect time and read timeout based on their application
+
+#### Introduce timeout configuration to AsyncSink
+
+##### [FLINK-35435](https://issues.apache.org/jira/browse/FLINK-35435)
+
+In Flink 1.20, We have introduced timeout configuration to `AsyncSink` with `retryOnTimeout`
+and `failOnTimeout` mechanisms to ensure the writer doesn't block on un-acknowledged requests.
+
+### Formats
+
+#### Allow set ReadDefaultValues to false for non-primitive types on Proto3
+
+##### [FLINK-33817](https://issues.apache.org/jira/browse/FLINK-33817)
+
+Protobuf format now supports set `protobuf.read-default-values = false` for non-primitive types on pb3.
+Since the default value for `protobuf.read-default-values` is `false`, this is a breaking change, 
+if you are using pb3, you can set 'protobuf.read-default-values' = 'true' to keep the original behavior.
+
+### REST API & Web UI
+
+#### Show JobType on WebUI
+
+##### [FLINK-29481](https://issues.apache.org/jira/browse/FLINK-29481)
+
+We display the JobType on the Flink WebUI. The job maintainer or platform administrator can easily
+see whether the Flink Job is running in Streaming or Batch Mode, which is useful for troubleshooting.
+
+#### Expose SlotId / SlotSharingGroup in Rest API
+
+##### [FLINK-29481](https://issues.apache.org/jira/browse/FLINK-29481)
```

Comment:
This should be a copy&paste error. The content is exactly the same as the content at line 263